docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>Fantastic explanation. </p><p> </p><p>I'm also considering the comment below the figure. I would add in one case we're visualizing the posterior of a parameter (pps=False) and in the other case we're visualizing the posterior predictive which is over the response variable. As you say, it incorporates  an additional source of uncertainty. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Thanks! And I have added this except instead of saying "posterior of a parameter" to "mean outcome parameter" to avoid the unambiguous meaning of "parameter". This way, we stay in line with the docstring of plot_cap (target="mean") and the GLM explanation in the beginning sections.</p>